spidermon/contrib/actions/email/smtp.py

```diff
@@ -0,0 +1,62 @@
+from __future__ import absolute_import
+import smtplib
+
+from spidermon.exceptions import NotConfigured
+
+from . import SendEmail
+
+
+class SendSmtpEmail(SendEmail):
+    smtp_host = "smtp.gmail.com"
+    smtp_port = 587
+    smtp_user = None
+    smtp_password = None
+    
+    def __init__(
+        self,
+        smtp_host=None,
+        smtp_port=None,
+        smtp_user=None,
+        smtp_password=None,        
+        *args,
+        **kwargs
+    ):
+        super(SendSmtpEmail, self).__init__(*args, **kwargs)
+        self.smtp_host = smtp_host or self.smtp_host
+        self.smtp_port = smtp_port or self.smtp_port
+        self.smtp_user = smtp_user or self.smtp_user
+        self.smtp_password = smtp_password or self.smtp_password
+
+        if not self.fake and not self.smtp_user:
+            raise NotConfigured("You must provide the smtp user.")
+        if not self.fake and not self.smtp_password:
+            raise NotConfigured("You must provide the smtp password.")
+
+    @classmethod
+    def from_crawler_kwargs(cls, crawler):
+        kwargs = super(SendSmtpEmail, cls).from_crawler_kwargs(crawler)
+        kwargs.update(
+            {
+                "smtp_host": crawler.settings.get("SPIDERMON_SMTP_HOST"),
+                "smtp_port": crawler.settings.get("SPIDERMON_SMTP_PORT"),
+                "smtp_user": crawler.settings.get("SPIDERMON_SMTP_USER"),
+                "smtp_password": crawler.settings.get("SPIDERMON_SMTP_PASSWORD"),
+            }
+        )
+        return kwargs
+
+    def send_message(self, message):       
+        if not self.sender:
+            raise NotConfigured("You must provide the sender.")
+        if not self.to:
+            raise NotConfigured("You must provide the receiver.")
+
+        if not self.sender or not self.to:
+            return
+
+        server = smtplib.SMTP(self.smtp_host, self.smtp_port)
```

Comment:
Sometimes using [smtplib.SMTP](https://docs.python.org/2/library/smtplib.html#smtplib.SMTP) is not appropriated and you need to use [smtplib.SMTP_SSL](https://docs.python.org/2/library/smtplib.html#smtplib.SMTP_SSL). It will be interesting to have an option so the user can define which one to use depending on the needs of the SMTP server being connected.

Reply:
Maybe it is worth checking https://docs.scrapy.org/en/latest/topics/email.html

Reply:
It makes sense. Using the built-in SMTP method from Scrapy would be a good solution.